packages/terra-list/src/Section.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListSectionHeader from './ListSectionHeader';
+import ListItem from './ListItem';
+
+const propTypes = {
+  /**
+   * The header content to be placed for display in a section.
+   */
+  headerContent: PropTypes.string,
+  /**
+   * The list of elements to be placed for display in a section.
+   */
+  listItems: PropTypes.array,
+};
+
+const defaultProps = {
+  headerContent: '',
+  listItems: [],
+};
+
+class Section extends React.Component {
+  constructor(props) {
+    super(props);
+    this.generateHeader = this.generateHeader.bind(this);
+  }
+
+  generateHeader() {
+    return <ListItem content={<ListSectionHeader id={this.props.headerContent} title={this.props.headerContent} onClick={this.handleClick} />} key={this.props.headerContent} />;
```

Comment:
`this.handleClick` is never defined

Reply:
Oh my bad, I guess, I didn't commit changes in my other branch and branched out to this one. I'll go ahead and remove this for now. This function should be defined in Accordion-Section-Headers branch. I think the same thing happened with test:jest script also. Sorry about this.